tools/automation/verify/verify_packages.py

```diff
@@ -48,7 +48,7 @@ def test_azure_cli_installation(self):
     def test_azure_cli_module_installation(self):
         expected_modules = set([n for n, _ in automation_path.get_command_modules_paths(include_prefix=True)])
 
-        installed_command_modules = [dist.key for dist in list(working_set) if dist.key.startswith(COMMAND_MODULE_PREFIX)]
+        installed_command_modules = [dist.metadata['Name'] for dist in distributions() if dist.metadata['Name'].startswith(COMMAND_MODULE_PREFIX)]
```

Comment:
`dist`'s type is `importlib.metadata.PathDistribution`, but this class is not documented by https://docs.python.org/3.15/library/importlib.metadata.html either.

Another way to access the name of the distribution is `dist.name`. Actually, `importlib.metadata.Distribution.name` is defined as

```py
    @property
    def name(self):
        """Return the 'Name' metadata for the distribution package."""
        return self.metadata['Name']
```